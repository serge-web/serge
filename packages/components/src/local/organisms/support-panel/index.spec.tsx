/* global it expect */
import React from 'react'
import renderer from 'react-test-renderer'
import SupportPanel from '.'

describe('Support Panel component: ', () => {
  it('renders component correctly', () => {
    const tree = renderer
      .create(<SupportPanel />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
