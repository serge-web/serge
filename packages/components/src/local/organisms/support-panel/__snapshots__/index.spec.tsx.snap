// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Support Panel component:  renders component correctly 1`] = `
<div
  className="root"
>
  <div
    className="panel"
    style={
      Object {
        "visibility": "hidden",
      }
    }
  >
    <div
      className="react-draggable"
      onMouseDown={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      style={
        Object {
          "boxSizing": "border-box",
          "cursor": "auto",
          "display": "flex",
          "flexShrink": 0,
          "height": "100%",
          "left": 0,
          "maxHeight": "100%",
          "maxWidth": "80%",
          "minHeight": "100%",
          "minWidth": 330,
          "position": "unset",
          "top": 0,
          "transform": "translate(0px,0px)",
          "userSelect": "auto",
          "width": "330px",
        }
      }
      transform={null}
    >
      <div
        className="content"
      >
        <div
          className="tab-panel"
          hidden={false}
          value="my_force"
        >
          My Force
        </div>
        <div
          className="tab-panel"
          hidden={true}
          value="my_orders"
        >
          <div
            className="messages-list"
          >
            Render planning messages in table here. Message count:
            0
          </div>
        </div>
        <div
          className="tab-panel"
          hidden={true}
          value="opfor"
        >
          OPFOR
        </div>
        <div
          className="resize-indicator-container"
        >
          <div
            className="resize-indicator-icon"
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root MuiSvgIcon-colorPrimary MuiSvgIcon-fontSizeLarge"
              focusable="false"
              style={
                Object {
                  "marginLeft": 0,
                }
              }
              viewBox="0 0 24 24"
            >
              <path
                d="M12 8c1.1 0 2-.9 2-2s-.9-2-2-2-2 .9-2 2 .9 2 2 2zm0 2c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm0 6c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2z"
              />
            </svg>
          </div>
        </div>
      </div>
      <div>
        <div
          className=""
          onMouseDown={[Function]}
          onTouchStart={[Function]}
          style={
            Object {
              "cursor": "row-resize",
              "height": "10px",
              "left": "0px",
              "position": "absolute",
              "top": "-5px",
              "userSelect": "none",
              "width": "100%",
            }
          }
        />
        <div
          className=""
          onMouseDown={[Function]}
          onTouchStart={[Function]}
          style={
            Object {
              "cursor": "col-resize",
              "height": "100%",
              "position": "absolute",
              "right": "-5px",
              "top": "0px",
              "userSelect": "none",
              "width": "10px",
            }
          }
        />
        <div
          className=""
          onMouseDown={[Function]}
          onTouchStart={[Function]}
          style={
            Object {
              "bottom": "-5px",
              "cursor": "row-resize",
              "height": "10px",
              "left": "0px",
              "position": "absolute",
              "userSelect": "none",
              "width": "100%",
            }
          }
        />
        <div
          className=""
          onMouseDown={[Function]}
          onTouchStart={[Function]}
          style={
            Object {
              "cursor": "col-resize",
              "height": "100%",
              "left": "-5px",
              "position": "absolute",
              "top": "0px",
              "userSelect": "none",
              "width": "10px",
            }
          }
        />
        <div
          className=""
          onMouseDown={[Function]}
          onTouchStart={[Function]}
          style={
            Object {
              "cursor": "ne-resize",
              "height": "20px",
              "position": "absolute",
              "right": "-10px",
              "top": "-10px",
              "userSelect": "none",
              "width": "20px",
            }
          }
        />
        <div
          className=""
          onMouseDown={[Function]}
          onTouchStart={[Function]}
          style={
            Object {
              "bottom": "-10px",
              "cursor": "se-resize",
              "height": "20px",
              "position": "absolute",
              "right": "-10px",
              "userSelect": "none",
              "width": "20px",
            }
          }
        />
        <div
          className=""
          onMouseDown={[Function]}
          onTouchStart={[Function]}
          style={
            Object {
              "bottom": "-10px",
              "cursor": "sw-resize",
              "height": "20px",
              "left": "-10px",
              "position": "absolute",
              "userSelect": "none",
              "width": "20px",
            }
          }
        />
        <div
          className=""
          onMouseDown={[Function]}
          onTouchStart={[Function]}
          style={
            Object {
              "cursor": "nw-resize",
              "height": "20px",
              "left": "-10px",
              "position": "absolute",
              "top": "-10px",
              "userSelect": "none",
              "width": "20px",
            }
          }
        />
      </div>
    </div>
    <div
      className="action-tab"
    >
      <p
        className="active"
        onClick={[Function]}
      >
        My Force
      </p>
      <p
        className=""
        onClick={[Function]}
      >
        My Orders
      </p>
      <p
        className=""
        onClick={[Function]}
      >
        OPFOR
      </p>
    </div>
  </div>
  <div
    className="action-tab secondary-action-tab"
  >
    <p
      className="active"
      onClick={[Function]}
    >
      My Force
    </p>
    <p
      className=""
      onClick={[Function]}
    >
      My Orders
    </p>
    <p
      className=""
      onClick={[Function]}
    >
      OPFOR
    </p>
  </div>
</div>
`;
